cassandane/Cassandane/Cyrus/Carddav.pm

```diff
@@ -1450,4 +1450,27 @@ EOF
     $self->assert_matches(qr/2500 members/, $value);
 }
 
+sub test_delete_default_addressbook
+    :min_version_3_9 :needs_component_httpd
```

Comment:
You've marked this for backporting as far as 3.6, so could you make this

```suggestion
    :min_version_3_6 :needs_component_httpd
```

... so that the test will run once it's backported?  Thanks